form/database/migrations/2024_01_16_122809_create_products_table.php

```diff
@@ -13,11 +13,11 @@
     public function up(): void  
     {
         Schema::create('products', function (Blueprint $table) {
-            $table->id();
-            $table->string('name');
-            $table->string('category');
-            $table->integer('price');
-            $table->integer('quantity');
+            $table->increments('id')->comment("Unique ID for each product");
+            $table->string('name',50)->comment("Name of the product");
```

Comment:
@Yashbanu is 50 and 10 enough for name and category ?

Reply:
@Naveen1895 I have given mine own set of category in blade file using radio button whose length is less than 10 so I restricted the length as 10. If necessary I can make a change on it.